CHANGES.md

```diff
@@ -71,7 +71,7 @@
 
 - The test suite for Coq now requires Coq >= 8.16 due to changes in the
   plugin loading mechanism upstream (which now uses findlib).
```

Comment:
```suggestion
  plugin loading mechanism upstream (which now uses Findlib).
```

Since Findlib is the name of the library manager, it should be capitalised. If the official way to write it (it's proper name) is findlib, please let me know, and I'll change it in my style guide.

I see it's written later as `findlib` as well. Is there a difference between the two, like one is the proper name of the library manager and `findlib` is a file or command. It appears " `findlib`-based loading method " is talking about the library manager itself, so would it be Findlib?

Reply:
I think the correct one is actually `Findlib` since it is the name of a library. The issue is that you almost never write it like this unless describing it in documentation since OCaml libraries typically are denoted with lowercase.

Reply:
Great! I'll update it in the style guide! Thank you!

Reply:
I think whatever we choose is right, however that's a tricky one, as the OPAM package name is `findlib` but the ML name is `Findlib` :)